docs/en/observability/create-alerts.asciidoc

```diff
@@ -3,14 +3,18 @@
 
 [IMPORTANT]
 ====
-Make sure to set up alerting in {kib}. For details, see
+Ensure Alerting is already set up in {kib}. For details, see
 {kibana-ref}/alerting-setup.html[Setup and prerequisites].
 ====
 
-Within the Logs, Metrics, and Uptime apps, alerting enables you to detect
-complex *conditions* defined by a *rule*. When a condition is met, the rule
+Alerting enables you to detect complex *conditions* defined by a rule
+within the Logs, Metrics, and Uptime apps. When a condition is met, the rule
```

Comment:
APM is RAC registered as well. We should added to the list as well.

Reply:
Will do. Thanks!

Reply:
@EamonnTP I have one more comment.  Since users can create rules either a) through the above mentioned apps (Logs, Metrics, Uptime and APM) or b) through Stack Management, do you think it is clear enough to the users that `not all predefined rules` in Stack Management will make generated alerts appear in the Alerts page? It happened to some users already that they created for example a Stack Monitoring Rule and they were expecting a new alert to appear in Alerts page. This was not a bug, it was expected, since Stack Monitoring rule types are not RAC registered yet. All I am saying is if it is clear enough which rule types will generate alerts for the Alerts page.

Reply:
Thanks @mgiota I added a note for more clarification in https://github.com/elastic/observability-docs/pull/1211/commits/8a7b49eeb612b0b592a4888a616658e54a8af892